test/qa/lib/__init__.py

```diff
@@ -274,10 +274,10 @@ def flush_allowlist():
 def is_connect_successful(output, name="", hostname=""):
     if name and hostname:
```

Comment:
If `name` and `hostname` are empty, wouldn't it be good to get them from the `output`? 
Then always check that they are the same for connecting and connected message, and always use
```
f"Connecting to {name} ({hostname})" in output
            and f"You are connected to {name} ({hostname})!" in output
```